lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -7275,6 +7275,7 @@ A tool can refer multiple reference IDs.
    <xrefs>
        <xref type="bio.tools">dada2</xref>
        <xref type="bioconductor">dada2</xref>
+       <xref type="biii">dada2</xref>
```

Comment:
dada2 is not correct here, isn't it?

Reply:
right. Wasn't aware that "dada" is an actual package, thought it was a placeholder. The example should be fixed in 055afb6.